conf/defaults.yaml

```diff
@@ -231,16 +228,13 @@ topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
 topology.shellbolt.max.pending: 100
 topology.skip.missing.kryo.registrations: false
 topology.max.task.parallelism: null
-topology.max.spout.pending: null
+topology.max.spout.pending: null  # TODO: We dont need this any more
 topology.state.synchronization.timeout.secs: 60
-topology.stats.sample.rate: 0.05
+topology.stats.sample.rate: 0.001
```

Comment:
Sure. I agree ... thats why I ask. Does 1 out of every 100 seem reasonable ? Or do you feel we should continue to sample 5 out of 100 ?  This change is based on impact observed during perf runs.... but if it -vely impacts operational convenience then worth reverting.